src/app/dashboard/dashboard.interfaces.ts

```diff
@@ -0,0 +1,6 @@
+import { IDevice } from '../devices/devices.interfaces';
+
+export interface IWidget {
```

Comment:
For example ```IWidget``` can be ```Widget```. It's accroding to [angular 2 naming convention](https://angular.io/docs/ts/latest/guide/style-guide.html#!#03-03)